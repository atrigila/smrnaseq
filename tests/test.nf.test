nextflow_pipeline {

    name "Test Workflow main.nf - test"
    script "main.nf"
    profile "test"
    tag "test"
    tag "pipeline"

    test("test") {

        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(UTILS.removeNextflowVersion("$outputDir")).match("software_versions") },
                { assert workflow.trace.succeeded().size() == 196 }
            )
        }

    }

}
