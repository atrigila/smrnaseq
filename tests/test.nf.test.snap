{
    "fastqc_trimmed": {
        "content": [
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:48.886414317"
    },
    "bowtie_index_genome": {
        "content": [
            "genome.2.ebwt:md5,8f007efb2d23e46035cc47340c4d981e",
            "genome.3.ebwt:md5,caf97e67055e1b465e933214d0168e68",
            "genome.1.ebwt:md5,1bfbf95848b5a2e8a786d0a2c1054bbb",
            "genome.rev.2.ebwt:md5,65f054c598e3e9fdd33cb22c9f8f8ca5",
            "genome.4.ebwt:md5,87ff9d3ef8b55f845a9205a19c729f1c",
            "genome.edited.fa:md5,5aaada29b1c575230fa1b7c3bd4d76db",
            "genome.rev.1.ebwt:md5,3275dafaca50d8cafde473b54e9ebbff"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:16.37660337"
    },
    "mirtrace": {
        "content": [
            "mirtrace-stats-contamination_basic.tsv:md5,a67910702b13918a7d53f1283119f899",
            "mirtrace-stats-length.tsv:md5,0609fcb0014f0d2af3308655b2418635",
            "mirtrace-stats-contamination_detailed.tsv:md5,feb096d380e1ae62b4e73d54ed79e891",
            "mirtrace-stats-rnatype.tsv:md5,8a333ff54c189c4a5a175a619139a9ac",
            "mirtrace-stats-phred.tsv:md5,3f645df09c8eb314577cf6c4afda10f4",
            "mirtrace-stats-qcstatus.tsv:md5,5fdd73860d2df454e21593df68975459",
            "mirtrace-stats-mirna-complexity.tsv:md5,fce67408e9a9ab4a1a6f97e7e23ea2c8"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-26T14:47:45.013663624"
    },
    "mirna_quant_reference": {
        "content": [
            "mature.fa_igenome.fa:md5,ef4cabcbea05adf334ba6891a50c951a",
            "hairpin.fa_igenome.fa:md5,5fb538f60d5288846600dba31010251e",
            "mature.fa_igenome.fa_idx.fa:md5,ef4cabcbea05adf334ba6891a50c951a",
            "hairpin.fa_igenome.fa_idx.fa:md5,482b0327b0c1e36748c1ee3d3be9be77"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:18.050822535"
    },
    "software_versions": {
        "content": [
            "{BOWTIE_MAP_GENOME={bowtie=1.3.1, samtools=1.14}, BOWTIE_MAP_HAIRPIN={bowtie=1.3.1, samtools=1.14}, BOWTIE_MAP_MATURE={bowtie=1.3.1, samtools=1.14}, BOWTIE_MAP_SEQCLUSTER={bowtie=1.3.1, samtools=1.14}, FASTP={fastp=0.23.4}, FASTQC_RAW={fastqc=0.12.1}, FASTQC_TRIM={fastqc=0.12.1}, FORMAT_HAIRPIN={fastx_toolkit=0.0.14}, FORMAT_MATURE={fastx_toolkit=0.0.14}, INDEX_GENOME={bowtie=1.3.1}, INDEX_HAIRPIN={bowtie=1.3.1}, INDEX_MATURE={bowtie=1.3.1}, MIRTOP_QUANT={mirtop=0.4.25}, MIRTRACE_RUN={mirtrace=1.0.1}, PARSE_HAIRPIN={seqkit=2.6.1}, PARSE_MATURE={seqkit=2.6.1}, SAMTOOLS_FLAGSTAT={samtools=1.2}, SAMTOOLS_IDXSTATS={samtools=1.2}, SAMTOOLS_INDEX={samtools=1.2}, SAMTOOLS_SORT={samtools=1.2}, SAMTOOLS_STATS={samtools=1.2}, SEQCLUSTER_SEQUENCES={seqcluster=1.2.9}, TABLE_MERGE={r-base=3.6.2}, Workflow={nf-core/smrnaseq=v2.3.2dev}}"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:16.357261443"
    },
    "mirna_quant_bam": {
        "content": [
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            "Clone9_N1_mature.sorted.flagstat:md5,014b846757652783ff3f954a966b8a1b",
            "Clone1_N1_mature.sorted.idxstats:md5,77ed64ddd23cf07340b65c04559d4726",
            "Clone9_N3_mature.sorted.stats:md5,23b012dd60ea6dcec5cedb5a73ffc792",
            "Clone9_N3_mature.sorted.flagstat:md5,4f60cfd3527e91df204253f9f1aebfcd",
            "Clone1_N1_mature.sorted.flagstat:md5,f138a532bc7dab94f7cf4b4a4e5e23e5",
            "Control_N1_mature.sorted.flagstat:md5,16318a574fe410b81c0e2cdf0dfa3c97",
            "Clone9_N3_mature.sorted.idxstats:md5,dee54e8e22dfca24acc290bf89a05cd4",
            "Clone1_N3_mature.sorted.flagstat:md5,67fc6ba8516d4e5b3ed2a62febc92f0a",
            "Clone9_N2_mature.sorted.idxstats:md5,c9643d610de2f4d449a876e41dcab872",
            "Clone9_N1_mature.sorted.idxstats:md5,c6eb22f3ff675e422d065375279150b2",
            "Control_N2_mature.sorted.idxstats:md5,12388de965d69b6102b62bc9cdf27e85",
            "Control_N2_mature.sorted.flagstat:md5,c8742d1b49535ff815c2def66cba681e",
            "Clone9_N2_mature.sorted.flagstat:md5,247e02da50509fc1b3c5b35001922a8d",
            "Control_N3_mature.sorted.idxstats:md5,5fb7b46915043bd3c947a37afc3e8075",
            "Clone1_N3_mature.sorted.idxstats:md5,805f115ac4d9c8033908bafaba6c92fb",
            "Control_N2_mature.sorted.stats:md5,ef5935898e3ca45d898053af335d1dfb",
            "Control_N1_mature.sorted.idxstats:md5,847bc5c6ef44e1f9754e47e648a5ef98",
            "Control_N3_mature.sorted.flagstat:md5,2831f55bccbcf666d1c313ef377bcb30",
            "Control_N1_mature.sorted.stats:md5,b4ffb3a7623e32420def4d3470a326ab",
            "Control_N3_mature.sorted.stats:md5,ff9dab4d3220aa44cdc80ea2fc311823",
            "Clone1_N3_mature.sorted.stats:md5,e8feced949af44c8e5d31523b12122b9",
            "Clone1_N1_mature.sorted.stats:md5,9c42ffda181ca24ab076efc16fcfd4b3",
            "Clone9_N1_mature.sorted.stats:md5,02184884c17c8730f4df9e1a6144bbdc",
            "Clone9_N2_mature.sorted.stats:md5,fb8c403012481dafb895013c555f6d85",
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            "Clone1_N3_mature_hairpin.sorted.flagstat:md5,6f14ec4cd502c0eb213b1905b4568a76",
            "Clone1_N3_mature_hairpin.sorted.stats:md5,471db325d8ccd1fe787b5dd91dccf487",
            "Control_N3_mature_hairpin.sorted.stats:md5,2f2249c537451d1f72dfe298f16093ac",
            "Control_N2_mature_hairpin.sorted.stats:md5,1195dd51f77e338db83dc66476ac7c5f",
            "Control_N1_mature_hairpin.sorted.stats:md5,feb69ad2f8bbc527e39502a297f08abd",
            "Clone1_N1_mature_hairpin.sorted.stats:md5,bd02cb15e666054edecc89d392d72902",
            "Clone9_N2_mature_hairpin.sorted.idxstats:md5,04b1c0ea4f133b5ee9086eec1f4077e7",
            "Control_N3_mature_hairpin.sorted.idxstats:md5,2c44710ce63bdebb12d64c729ef338b0",
            "Clone9_N1_mature_hairpin.sorted.idxstats:md5,238b462a2418d545418c0b3a28029bdf",
            "Clone9_N1_mature_hairpin.sorted.flagstat:md5,3e4b295b54df05c438603edaa53123b1",
            "Control_N2_mature_hairpin.sorted.flagstat:md5,2605548b5ab1e6ad6b5cf88ea0030909",
            "Clone9_N3_mature_hairpin.sorted.idxstats:md5,9523d29d411c2efb0a1756355230fa98",
            "Control_N1_mature_hairpin.sorted.flagstat:md5,41769a400a176ea78432749f07f80fda",
            "Clone9_N2_mature_hairpin.sorted.flagstat:md5,663ae0bb8b29d1e1c3d75ef5df1015a4",
            "Clone1_N1_mature_hairpin.sorted.idxstats:md5,0bca2d01f734128f3c11cf2697433048",
            "Clone9_N2_mature_hairpin.sorted.stats:md5,7bb8654d44dac245aec6aa045250628a",
            "Clone9_N1_mature_hairpin.sorted.stats:md5,155f327a9be0afa9584ec426826b5457",
            "Clone1_N1_mature_hairpin.sorted.flagstat:md5,8d0c785fec7b300200863c68b158444c",
            "Control_N2_mature_hairpin.sorted.idxstats:md5,8859b6e432cef8b24d5fb258a06aaabe",
            "Clone1_N3_mature_hairpin.sorted.idxstats:md5,5b6cd8d5784c387d2649f18499a17c6e",
            "Clone9_N3_mature_hairpin.sorted.flagstat:md5,efc65cb620ca139b19967e12f13ba007",
            "Control_N1_mature_hairpin.sorted.idxstats:md5,e1f455195f1cb42a1d1e58770aaeb2b4",
            "Clone9_N3_mature_hairpin.sorted.stats:md5,9f61c028bb1c92667e31093156ada662",
            "Control_N3_mature_hairpin.sorted.flagstat:md5,f3df866b94dc6ef355b01f22047aaf4f"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:26.750655347"
    },
    "mirna_quant_edger_qc": {
        "content": [
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:39.507346289"
    },
    "fastqc_raw": {
        "content": [
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:43.59673405"
    },
    "multiqc_multiqc_data": {
        "content": [
            "fastqc-status-check-heatmap.txt:md5,30f55902420433a11971d36956e8576e",
            "fastp_filtered_reads_plot.txt:md5,0af4d3cfbf463f33b2b88cdfd69909ac",
            "fastqc_overrepresented_sequences_plot.txt:md5,d861a4a956a3023efa19b1259d8665b2",
            true,
            "fastqc_sequence_counts_plot-1.txt:md5,37d8ae6c5734eb2679e65e46c9af9101",
            "mirtrace_complexity_plot.txt:md5,bb92ed0659dddb15d3e01686acef85b5",
            "fastqc_per_sequence_gc_content_plot-1_Percentages.txt:md5,edf5130b48b0814788867dd42e9cf407",
            "multiqc_citations.txt:md5,3adbccd17a42d0d5d97ee7ebb476f433",
            true,
            "fastqc_sequence_length_distribution_plot.txt:md5,13f72b3be854c74cd7b5176ca0583770",
            "fastp-seq-content-n-plot_Read_1_Before_filtering.txt:md5,048e9b2293217ab8b47ba0113b83c76c",
            "fastqc_sequence_duplication_levels_plot-1.txt:md5,06a4162d4787883b1f39a1b9acb7bb91",
            "fastqc_per_base_sequence_quality_plot-1.txt:md5,6a0df664a369ee216412f0033ed3a042",
            true,
            "fastqc_per_base_n_content_plot-1.txt:md5,048e9b2293217ab8b47ba0113b83c76c",
            "fastqc_per_base_n_content_plot.txt:md5,fb8a49467d1679dbf716e1f00c1f25e5",
            "fastp-seq-quality-plot_Read_1_After_filtering.txt:md5,b1481a9fbf5ea541b39aed99b51d616f",
            "fastqc_per_sequence_quality_scores_plot.txt:md5,38dbd6509de7f1beac17e44acb8336be",
            "mirtrace_qc_plot.txt:md5,9036243402f6248214dc0458cb1291e9",
            "fastqc_per_sequence_quality_scores_plot-1.txt:md5,69f4459318311103fe71a5d1d3b2c076",
            "mirtrace_length_plot.txt:md5,86a0d54324cbe4817b112b02d4872cbb",
            true,
            "fastqc_overrepresented_sequences_plot-1.txt:md5,1de000de50cc7516bb9d996def14c6eb",
            "fastqc-status-check-heatmap-1.txt:md5,e64c353797e6dc057d10b3194ca6b20d",
            "fastqc_sequence_counts_plot.txt:md5,384516be5885fdeac59e70c17f1413c2",
            "mirtrace_rna_categories_plot.txt:md5,9a57e6b3a3dd75906134030d25f6a7a9",
            "fastp-seq-quality-plot_Read_1_Before_filtering.txt:md5,65685440fcb927115ac30b01133e2f92",
            true,
            "fastqc_per_base_sequence_quality_plot.txt:md5,80ea53d94917cadcb41f3fdb56cba014",
            true,
            "fastp-seq-content-n-plot_Read_1_After_filtering.txt:md5,048e9b2293217ab8b47ba0113b83c76c",
            true,
            true,
            "fastqc_adapter_content_plot.txt:md5,86a0f478fd3e690ef5132d08bf4e7625",
            "fastqc_sequence_duplication_levels_plot.txt:md5,88b0babe77f623547e8af664791ef2d0",
            "fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,539883757ae5bf731da93fe0568022e9",
            "fastqc_per_sequence_gc_content_plot-1_Counts.txt:md5,ed7205b9c46d5f431c5b3bf499a673a1",
            "fastqc_adapter_content_plot-1.txt:md5,71ba0983a71f3a833526ab83a985d072",
            "fastp-seq-content-gc-plot_Read_1_Before_filtering.txt:md5,a8387a3e6b350acd16bb0ed5a43949d8",
            "fastqc_per_sequence_gc_content_plot_Counts.txt:md5,3e9c1a7204c12d9c1b3b53342cf61840",
            true,
            "fastp-seq-content-gc-plot_Read_1_After_filtering.txt:md5,efa8123c9ad43f99a6fba671f50322ca"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-26T14:48:43.985133435"
    },
    "mirna_quant_seqcluster": {
        "content": [
            "Control_N2.fastp_trimmed.fastq.gz:md5,d6e607edcb3d71c546ce8fdfb18e3999",
            "Clone9_N3.fastp_trimmed.fastq.gz:md5,67cf9a7d2fa431f0e772391ed1651f93",
            "Clone1_N3.fastp_trimmed.fastq.gz:md5,2e395240c74b3870efe03dc329d8bad8",
            "Control_N1.fastp_trimmed.fastq.gz:md5,2f09082eb68a8f1dc308050e330887ec",
            "Clone9_N2.fastp_trimmed.fastq.gz:md5,c922ec95a4b84d806dd2054ff0f9cab7",
            "Clone9_N1.fastp_trimmed.fastq.gz:md5,115cf0c346a93a47ba782c5724b4f090",
            "Clone1_N1.fastp_trimmed.fastq.gz:md5,14a2590e663dbde105780726c2173c25",
            "Control_N3.fastp_trimmed.fastq.gz:md5,342368a6e6acdaf0ae9379676b6d07f0"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:18.37847702"
    },
    "mirtrace_qc_passed_reads.rnatype_unknown.collapsed": {
        "content": [
            "Clone1_N1.fastp.fasta:md5,ba5a705fbc7c4aadd259c29c3a443d97",
            "Clone9_N1.fastp.fasta:md5,3e9a4ce784cbcdb946a3e271cc899da9",
            "Control_N3.fastp.fasta:md5,850c3a525359f14ea488fcd0eedc2e36",
            "Control_N1.fastp.fasta:md5,446f3452ffac337eb1afa5861c2ba94a",
            "Clone9_N2.fastp.fasta:md5,168aa25977927be9480f9fe2399a82c7",
            "Control_N2.fastp.fasta:md5,5018f8b15dbd524eb7dd0ce39e9b7101",
            "Clone1_N3.fastp.fasta:md5,7a6c3163b377b25cf2886004ea9d4a73",
            "Clone9_N3.fastp.fasta:md5,80a4b70ef276cf5628dfa2942c7e4988"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-26T14:48:32.628512644"
    },
    "genome_quant_bam": {
        "content": [
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            true,
            "Clone9_N2_mature_hairpin_genome.sorted.flagstat:md5,b8fbbdaca0458b19732312dea2997be8",
            "Control_N3_mature_hairpin_genome.sorted.stats:md5,53e8fca566dd5dd8553b25c38e961f6b",
            "Clone1_N1_mature_hairpin_genome.sorted.flagstat:md5,4d552770e6d3b06775b6f484684f7a77",
            "Clone9_N1_mature_hairpin_genome.sorted.stats:md5,f819db926f570a9bb1cf9b6adff7d758",
            "Clone9_N1_mature_hairpin_genome.sorted.flagstat:md5,2fd8cd41edde2292f857528658c3e0a1",
            "Clone1_N1_mature_hairpin_genome.sorted.stats:md5,64b19984a4c2fa55e69a28fe7c138d7c",
            "Clone1_N3_mature_hairpin_genome.sorted.flagstat:md5,9dee041c82ad3b2d2ab4f8986822f2ce",
            "Control_N1_mature_hairpin_genome.sorted.flagstat:md5,18f0028f1cfc137ef8f089e465fe20f5",
            "Control_N3_mature_hairpin_genome.sorted.flagstat:md5,ae834e3dbaa18aa9bce66c3b0034c430",
            "Clone9_N1_mature_hairpin_genome.sorted.idxstats:md5,a199f8999ba915a81acfc450e0db8393",
            "Clone9_N2_mature_hairpin_genome.sorted.stats:md5,6db5d451fa95a77247d38c4a263cf05d",
            "Clone9_N3_mature_hairpin_genome.sorted.idxstats:md5,0a227abfa6aa10023219bce7b2795b67",
            "Control_N2_mature_hairpin_genome.sorted.stats:md5,f105861208044e44644c74449d0c6142",
            "Control_N1_mature_hairpin_genome.sorted.idxstats:md5,fe01b41ed88612c28b667817c4a8fcc1",
            "Control_N2_mature_hairpin_genome.sorted.idxstats:md5,b6da32ce230fdeab39f3c4d5dcf41a3a",
            "Control_N1_mature_hairpin_genome.sorted.stats:md5,b2afcfcb9e785a71bb5fce25b87fd67a",
            "Control_N2_mature_hairpin_genome.sorted.flagstat:md5,b0d42e94b2e0e026cd32a88dce40722c",
            "Clone9_N3_mature_hairpin_genome.sorted.stats:md5,8ec73033ee0e2ad16fdc497b209adb50",
            "Clone9_N2_mature_hairpin_genome.sorted.idxstats:md5,61c4821cba66c91272e62547617db2c9",
            "Clone1_N1_mature_hairpin_genome.sorted.idxstats:md5,169a196ee2da6945392f6a361164ffc5",
            "Clone1_N3_mature_hairpin_genome.sorted.stats:md5,fb089da3351a3f8c0bc742f3f4533349",
            "Control_N3_mature_hairpin_genome.sorted.idxstats:md5,94d53c9ad335779a1ea25fd5b4b08069",
            "Clone1_N3_mature_hairpin_genome.sorted.idxstats:md5,d680e48370d319f1891cdd61b8754644",
            "Clone9_N3_mature_hairpin_genome.sorted.flagstat:md5,dbe38679a68d55b8a182831e336eebb1"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:52:20.338493953"
    },
    "bowtie_index_mirna_mature": {
        "content": [
            "fasta_bidx.3.ebwt:md5,7aeccc6ec919ce1cfcd2b1e40be743fe",
            "fasta_bidx.2.ebwt:md5,778bd05aa462c2119736f9a4056bc41f",
            "fasta_bidx.1.ebwt:md5,11a2f568aa785d32246cb8ce1aa8add4",
            "fasta_bidx.4.ebwt:md5,78260e9dde22d8f31622c77d71bbb605",
            "fasta_bidx.rev.2.ebwt:md5,ab52bc362b4c382cf213a71219e9efee",
            "fasta_bidx.rev.1.ebwt:md5,0520d43b6b75328f3c30254a2d6baa18"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:16.994060474"
    },
    "bowtie_index_mirna_hairpin": {
        "content": [
            "fasta_bidx.1.ebwt:md5,40c2c142e094ab5ea531a0564ad62cee",
            "fasta_bidx.2.ebwt:md5,6d99166d1a8a7a1bbf262e89db218278",
            "fasta_bidx.3.ebwt:md5,5c60180e2f6fc2e0c399f10b9f9bbee2",
            "fasta_bidx.4.ebwt:md5,c3d4dbfbb587261ddf29e906029c10b1",
            "fasta_bidx.rev.1.ebwt:md5,4532fb1426b29163b23702c8daf84d62",
            "fasta_bidx.rev.2.ebwt:md5,4270939b810aa3f461bacee1edc993aa"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:17.519448134"
    },
    "mirtrace_qc_passed_reads.all.collapsed": {
        "content": [
            "Clone1_N1.fastp.fasta:md5,3761badda2e1c2dd2da3eb1bbfe6e1f8",
            "Clone9_N1.fastp.fasta:md5,97310b5094e02329f412bae2657a2765",
            "Control_N3.fastp.fasta:md5,f24ddebb1b012ec3817b88ff2e8989ea",
            "Control_N1.fastp.fasta:md5,888c2e9639dd152d2f63e3c00019376b",
            "Clone9_N2.fastp.fasta:md5,7f3879545cf3beb43af71ce83430abbf",
            "Control_N2.fastp.fasta:md5,16e03efca3b17ec279658ad1556f52e4",
            "Clone1_N3.fastp.fasta:md5,c6f029e821dceb6d60869a8f3e6e8198",
            "Clone9_N3.fastp.fasta:md5,43d45f4416d0028f9adadd0f5ca44d9c"
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-26T14:48:10.730193602"
    },
    "mirna_quant_mirtop": {
        "content": [
            "mirtop_rawData.tsv:md5,774410dd2cf1e5be83bf8d6e1e8b96b0",
            true,
            true,
            "mirna.tsv:md5,1f840afe7ab68eac73feaaad321b64f6",
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-08-22T22:51:32.899870265"
    }
}
